features/in_onderzoek.feature

```diff
@@ -166,188 +165,265 @@ Functionaliteit: in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                     | waarde              |
+      | locatiebeschrijving (12.10)              | locatiebeschrijving |
+      | aanduiding gegevens in onderzoek (83.10) | <waarde>            |
+      | datum ingang onderzoek (83.20)           | 20220307            |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 555550001                           |
+      | fields              | verblijfplaats.verblijfadres.<veld> |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde              |
+      | type                                                       | Locatie             |
+      | verblijfadres.locatiebeschrijving                          | locatiebeschrijving |
+      | verblijfadres.inOnderzoek.<veld>                           | true                |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07          |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum               |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022        |
+
+      Voorbeelden:
+      | waarde | veld                |
+      | 081210 | locatiebeschrijving |
+
+    Abstract Scenario: persoon heeft <veld> van verblijfplaats in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                     | waarde                |
+      | land adres buitenland (13.10)            | 1234                  |
+      | aanduiding gegevens in onderzoek (83.10) | <in onderzoek waarde> |
+      | datum ingang onderzoek (83.20)           | 20220307              |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 555550001                           |
+      | fields              | verblijfplaats.verblijfadres.<veld> |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde                   |
+      | type                                                       | VerblijfplaatsBuitenland |
+      | verblijfadres.<veld>.code                                  | <waarde>                 |
```

Comment:
dit werkt niet bij <veld> regel3

Reply:
Tja, dit komt door het toevoegen van hacks in de automation code (als waarde is lege string, insert veld dan niet in expected json), waardoor de fout in de Dan stap wordt weggefilterd. Scenario is nu gesplitst in 2 scenario's